### PR TITLE
Added the scikit-learn requirements of the folder structure

### PR DESCRIPTION
**Pull request by krisztinaknagy**
Added the requirements of the folder structure when using the provider in Pulse.

### CODE REVIEW

File: openml-scikit/README.md [hunk 1]
```diff
@@ -39,3 +39,7 @@ When running the java process that will use this provider, be sure to have confi
 ```bash
 export JAVA_LIBRARY_PATH=/usr/local/anaconda3/envs/scikit-env-test/lib
 ```
+
+### Usage
+
+When the user [imports a model](https://docs.feedzai.com/display/pulse/Importing+an+external+model) to Pulse using this provider, the import assumes a folder called ```model```, containing a pickle file (.pkl).
```
Comment (pedrorijo91) [inline openml-scikit/README.md]: this link is not public, and as such it doesn't make sense to be linked from this public repository

Comment (pedrorijo91) [inline openml-scikit/README.md]: probably Pulse is not known to readers. Maybe call it "Feedzai platform" is more understandable

**Review by pedrorijo91 — CHANGES_REQUESTED** [review]
remove the link to private Feedzai pages

**Review by pedrorijo91 — APPROVED** [review]
nice :) will merge soon if no one has anything more to say on this

**Review by pedrorijo91 — APPROVED** [review]

File: openml-scikit/README.md [hunk 2]
```diff
@@ -39,3 +39,7 @@ When running the java process that will use this provider, be sure to have confi
 ```bash
 export JAVA_LIBRARY_PATH=/usr/local/anaconda3/envs/scikit-env-test/lib
 ```
+
+### Usage
+
+When the user imports a model to the Feedzai platform using this provider, the import assumes a folder called ```model```, containing a pickle file (.pkl).
```
Comment (nmldiegues) [inline openml-scikit/README.md]: I usually like to give examples like Unix "tree" command.
E.g.: 

(don't use this *literal* example :) )

nuno.diegues@Laptop-511:~/workspace/feedzai-openml-python/openml-scikit/src/test/resources$ tree
.
├── no_probability
│   └── model
│       └── svc_no_proba.joblib.pkl
├── rf
│   └── model
│       └── svc__rf_proba.joblib.pkl
└── svm
    ├── model
    │   └── svc.joblib.pkl
    └── model.json

Comment (nmldiegues) [reply openml-scikit/README.md]: Well, the indentation is not working great, so let's try with a screenshot:
image

**Review by pedrorijo91 — APPROVED** [review]

**Comment by nmldiegues** [comment]
Merged and cherry picked to hf-0.1.X

**Pull request merged**